src/manifest.json

```diff
@@ -3,9 +3,14 @@
       "cockpit": "215"
   },
 
-  "tools": {
+  "menu": {
       "index": {
-          "label": "Navigator"
+          "label": "File browser",
+	  "keywords": [
+		{
+			"matches": ["files", "navigator"]
```

Comment:
some more come to my mind: filesystem, file, directory, directories, permissions (feel free to pick or ignore)

Reply:
filesystem feels good, directory can, permissions is a bit vague if you ask me

Reply:
Did you want to adjust these, or shall we just merge with what we have?

I might add "explorer" for the Windows users...

Reply:
Adjusting